.github/workflows/tools-releases.yml

```diff
@@ -0,0 +1,38 @@
+name: Kubernetes Tools Releases for controller-runtime envtest
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        kubernetes_version: [ 1.28.0 ]
+        os: [ linux darwin windows ]
+        arch: [ amd64 arm64 ]
+        include:
+          - arch: ppc64le
+            os: linux
+          - arch: s390x
+            os: linux
```

Comment:
I think this combinations won't get kubernetes_version (if I read this here correctly: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)